sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/Lease.java

```diff
@@ -35,20 +52,36 @@ public interface Lease {
      */
     String getOwner();
 
+    /**
+     * Gets the version of this lease. See {@link LeaseVersion} for more details.
+     * @return version of the lease.
+     */
+    LeaseVersion getVersion();
+
+    /**
+     * Sets the lease version for the lease.
+     * @param leaseVersion lease version for the lease.
+     */
+    void setVersion(LeaseVersion leaseVersion);
+
+    /**
+     * Sets the feed range for the lease.
+     * @param feedRangeInternal feed range for the lease.
+     */
+    void setFeedRange(FeedRangeInternal feedRangeInternal);
+
     /**
      * Gets the timestamp of the lease.
      *
      * @return the timestamp of the lease.
      */
     String getTimestamp();
 
-    ChangeFeedState getIncrementalContinuationState(
+    ChangeFeedState getContinuationState(
         String containerRid,
         FeedRangeInternal feedRange);
 
-    ChangeFeedState getFullFidelityContinuationState(
-        String containerRid,
-        FeedRangeInternal feedRange);
+    ChangeFeedState getContinuationStateV1(String containerRid);
```

Comment:
does V1 = FullFidelity or V1 = newlkeaseformat? 

Since we always use ChangeFeedMode.FULL_FIDELITY in this method, so feels like getFullFidelityContinuationState is better

Reply:
using LeaseVersion enum names in implementation detail will improve the readability here.